test/lib/sdkdriver/src/util/index.ts

```diff
@@ -31,10 +31,11 @@ export function createWebDriver(browser: 'chrome' | 'firefox'): WebDriver {
 
 /**
  * Create an express web server that serves the SDK setup code.
+ * @param {string} rootPath - Root folder of the web server
  * @returns {HTTPServer}
  */
-export function createWebServer(): HTTPServer {
+export function createWebServer(rootPath: string): HTTPServer {
   const app: express.Application = express();
-  app.use(express.static(join(__dirname, '..', '..', 'browser')));
+  app.use(express.static(rootPath));
```

Comment:
OK, so the idea is that we pass the `rootPath` and serve things up via Express `static` middleware, not unlike how we do it in our E2E tester? And so, at build time, we'll want to Browserify the necessary modules?